src/lib/dnssd/minimal_mdns/Server.cpp

```diff
@@ -386,7 +386,9 @@ CHIP_ERROR ServerBase::BroadcastImpl(chip::System::PacketBufferHandle && data, u
             }
             else
             {
-                ChipLogError(Discovery, "Attempt to mDNS broadcast failed:  %s", chip::ErrorStr(err));
+                char ifaceName[20];
+                info->mInterfaceId.GetInterfaceName(ifaceName, sizeof(ifaceName));
+                ChipLogDetail(Discovery, "Warning: Attempt to mDNS broadcast failed on %s:  %s", ifaceName, chip::ErrorStr(err));
```

Comment:
use err.Format() or err.AsString() (Format is prefferable for small platforms to support %d ... be sure to use the macro) instead of ChipErrorStr

Reply:
That portion was existing code, but I can update it.